about/complying_with_licenses.rst

```diff
@@ -93,7 +93,7 @@ Here's a list of libraries requiring attribution:
 FreeType
 ^^^^^^^^
 
-Godot uses `FreeType <https://www.freetype.org/>`_ to render fonts. Its license
+Godot uses `FreeType <https://www.freetype.org/>`_ to render :ref:`DynamicFonts <class_DynamicFont>`. Its license
```

Comment:
DynamicFont doesn't exist anymore in 4.0 so it's not a good change for `master`. Maybe it should spell "TTF and OTF fonts" instead? @bruvzg

While at it, please try to keep the manual wrapping consistent.

Reply:
I do not think it's necessary to specify it.
The default font is TTF (https://github.com/godotengine/godot/pull/39603).
And even if it's not used, FreeType is compiled in and attribution is required anyway.

Reply:
I agree. Users can build Godot without freetype if they want but at this stage they should be aware of what they're doing and this documentation tweak wouldn't matter much.